Carthage support please

Unable to build with Carthage.
This is Build error log.

```
*** Building scheme "RxKeyboard" in RxKeyboard.xcworkspace
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Script-E1B117AD07E9F1CD1EB57A47.sh
	CompileSwift normal arm64 /Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
	CompileSwift normal armv7 /Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Headers/RxKeyboard-Swift.h /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard-Swift.h
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Headers/RxKeyboard-Swift.h /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard-Swift.h
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm.swiftmodule /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard.swiftmodule
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm64.swiftmodule /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard.swiftmodule
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm64.swiftdoc /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard.swiftdoc
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm.swiftdoc /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard.swiftdoc
	Ld /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard normal armv7
	Ld /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard normal arm64
	CreateUniversalBinary /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/RxKeyboard normal armv7\ arm64
	GenerateDSYMFile /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework.dSYM /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/RxKeyboard
	PhaseScriptExecution [CP]\ Copy\ Pods\ Resources /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Script-4896D7AF381BA8888BCE1918.sh
(16 failures)
error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.
/Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift:11:8: error: no such module 'RxCocoa'
/Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift:11:8: error: no such module 'RxCocoa'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
clang: error: linker command failed with exit code 1 (use -v to see invocation)
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: can't open input file: /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard (No such file or directory)
error: cannot parse the debug map for "/Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/RxKeyboard": No such file or directory
A shell task (/usr/bin/xcrun xcodebuild -workspace /Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/RxKeyboard.xcworkspace -scheme RxKeyboard -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Script-E1B117AD07E9F1CD1EB57A47.sh
	CompileSwift normal arm64 /Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
	CompileSwift normal armv7 /Users/muukii/Develop/pairs-gl/Carthage/Checkouts/RxKeyboard/Sources/RxKeyboard.swift
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Headers/RxKeyboard-Swift.h /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard-Swift.h
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Headers/RxKeyboard-Swift.h /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard-Swift.h
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm.swiftmodule /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard.swiftmodule
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm64.swiftmodule /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard.swiftmodule
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm64.swiftdoc /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard.swiftdoc
	Ditto /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/Modules/RxKeyboard.swiftmodule/arm.swiftdoc /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard.swiftdoc
	Ld /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/armv7/RxKeyboard normal armv7
	Ld /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Objects-normal/arm64/RxKeyboard normal arm64
	CreateUniversalBinary /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/RxKeyboard normal armv7\ arm64
	GenerateDSYMFile /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework.dSYM /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Products/Release-iphoneos/RxKeyboard.framework/RxKeyboard
	PhaseScriptExecution [CP]\ Copy\ Pods\ Resources /Users/muukii/Library/Developer/Xcode/DerivedData/RxKeyboard-hbcsxjhrandpghgqlrrpgrqlidsw/Build/Intermediates/RxKeyboard.build/Release-iphoneos/RxKeyboard.build/Script-4896D7AF381BA8888BCE1918.sh
(16 failures)
```

--- a/.gitignore
+++ b/.gitignore
@@ -5,4 +5,5 @@
 *.mode1v3
 
 Pods/
+Carthage/
 vendor/

--- a/Cartfile
+++ b/Cartfile
@@ -1,0 +1,1 @@
+github "ReactiveX/RxSwift" "3.0.0-beta.2"

--- a/Cartfile.resolved
+++ b/Cartfile.resolved
@@ -1,0 +1,1 @@
+github "ReactiveX/RxSwift" "3.0.0-beta.2"

--- a/RxKeyboard.xcodeproj/project.pbxproj
+++ b/RxKeyboard.xcodeproj/project.pbxproj
@@ -9,6 +9,8 @@
 /* Begin PBXBuildFile section */
 		0310C6741DA9525200BDA512 /* RxKeyboard.h in Headers */ = {isa = PBXBuildFile; fileRef = 0310C6711DA9525200BDA512 /* RxKeyboard.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		0310C6771DA9535200BDA512 /* RxKeyboard.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0310C6761DA9535200BDA512 /* RxKeyboard.swift */; };
+		034E7CA71DAD548900CFC398 /* RxCocoa.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 034E7CA31DAD548900CFC398 /* RxCocoa.framework */; };
+		034E7CA81DAD548900CFC398 /* RxSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 034E7CA41DAD548900CFC398 /* RxSwift.framework */; };
 		035BC8301DAADE86006CA9BE /* RxKeyboard.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 03A258CB1DA94EC50035A85B /* RxKeyboard.framework */; };
 		035BC8311DAADE86006CA9BE /* RxKeyboard.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 03A258CB1DA94EC50035A85B /* RxKeyboard.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		035F28D11DA95C050060CE31 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 035F28C71DA95C050060CE31 /* Assets.xcassets */; };
@@ -17,7 +19,6 @@
 		035F28D61DA95C580060CE31 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 035F28CB1DA95C050060CE31 /* AppDelegate.swift */; };
 		035F28D71DA95C5B0060CE31 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 035F28CD1DA95C050060CE31 /* ViewController.swift */; };
 		198DAF5A87143EF56B410376 /* Pods_RxKeyboard_RxKeyboardDemo.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1FD6156651B6B63C23AC292E /* Pods_RxKeyboard_RxKeyboardDemo.framework */; };
-		A51EC8A43DC72B5150E1DA0D /* Pods_RxKeyboard.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 387849BCC512BDBC37A69346 /* Pods_RxKeyboard.framework */; };
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
@@ -49,6 +50,10 @@
 		0310C6731DA9525200BDA512 /* Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		0310C6761DA9535200BDA512 /* RxKeyboard.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = RxKeyboard.swift; sourceTree = "<group>"; };
 		0310C67C1DA9572500BDA512 /* RxKeyboardDemo.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = RxKeyboardDemo.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		034E7CA21DAD548900CFC398 /* RxBlocking.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = RxBlocking.framework; path = Carthage/Build/iOS/RxBlocking.framework; sourceTree = "<group>"; };
+		034E7CA31DAD548900CFC398 /* RxCocoa.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = RxCocoa.framework; path = Carthage/Build/iOS/RxCocoa.framework; sourceTree = "<group>"; };
+		034E7CA41DAD548900CFC398 /* RxSwift.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = RxSwift.framework; path = Carthage/Build/iOS/RxSwift.framework; sourceTree = "<group>"; };
+		034E7CA51DAD548900CFC398 /* RxTests.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = RxTests.framework; path = Carthage/Build/iOS/RxTests.framework; sourceTree = "<group>"; };
 		035F28C71DA95C050060CE31 /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
 		035F28C91DA95C050060CE31 /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
 		035F28CB1DA95C050060CE31 /* AppDelegate.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = AppDelegate.swift; sourceTree = "<group>"; };
@@ -83,7 +88,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				A51EC8A43DC72B5150E1DA0D /* Pods_RxKeyboard.framework in Frameworks */,
+				034E7CA71DAD548900CFC398 /* RxCocoa.framework in Frameworks */,
+				034E7CA81DAD548900CFC398 /* RxSwift.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
@@ -183,6 +189,10 @@
 		221051FE9374C6EEB61B3418 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				034E7CA21DAD548900CFC398 /* RxBlocking.framework */,
+				034E7CA31DAD548900CFC398 /* RxCocoa.framework */,
+				034E7CA41DAD548900CFC398 /* RxSwift.framework */,
+				034E7CA51DAD548900CFC398 /* RxTests.framework */,
 				387849BCC512BDBC37A69346 /* Pods_RxKeyboard.framework */,
 				ED2E539A89FFC53BA70B139C /* Pods_Demo.framework */,
 				B1763C0DE08C3E15BD71C29F /* Pods_RxKeyboard_Demo.framework */,
@@ -246,12 +256,10 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 03A258D31DA94EC50035A85B /* Build configuration list for PBXNativeTarget "RxKeyboard" */;
 			buildPhases = (
-				E1B117AD07E9F1CD1EB57A47 /* [CP] Check Pods Manifest.lock */,
 				03A258C61DA94EC50035A85B /* Sources */,
 				03A258C71DA94EC50035A85B /* Frameworks */,
 				03A258C81DA94EC50035A85B /* Headers */,
 				03A258C91DA94EC50035A85B /* Resources */,
-				4896D7AF381BA8888BCE1918 /* [CP] Copy Pods Resources */,
 			);
 			buildRules = (
 			);
@@ -325,21 +333,6 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		4896D7AF381BA8888BCE1918 /* [CP] Copy Pods Resources */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			name = "[CP] Copy Pods Resources";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-RxKeyboard/Pods-RxKeyboard-resources.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
 		4FFBB804D69572B8C7099324 /* [CP] Copy Pods Resources */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
@@ -383,21 +376,6 @@
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-RxKeyboard-RxKeyboardDemo/Pods-RxKeyboard-RxKeyboardDemo-frameworks.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
-		E1B117AD07E9F1CD1EB57A47 /* [CP] Check Pods Manifest.lock */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			name = "[CP] Check Pods Manifest.lock";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
 			showEnvVarsInLog = 0;
 		};
 /* End PBXShellScriptBuildPhase section */
@@ -582,6 +560,10 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
+				);
 				INFOPLIST_FILE = "$(SRCROOT)/Supporting Files/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
@@ -604,6 +586,10 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
+				);
 				INFOPLIST_FILE = "$(SRCROOT)/Supporting Files/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";

